src/ecrc-frontend/src/components/page/informationreview/__snapshots__/InformationReview.test.js.snap

```diff
@@ -396,6 +396,8 @@ exports[`InformationReview Component Matches the snapshot 1`] = `
           </span>
         </label>
       </section>
+      <br />
```

Comment:
so in the snapshot, it doesnt seem to be rendering the share component
is this expected?

Reply:
Fixed